### PR TITLE
Improve helm and allow some parameters in command line

### PR DESCRIPTION
**Pull request by albundy83**
Add some improvements using helm chart.
Allow to pass parameters to command (especially `--skip-resource Oos`)

### CODE REVIEW

**Comment by outscale-vbr** [comment]
@albundy83 
Can you update the version of helm charts with your change ?

https://github.com/outscale/osc-cost/blob/main/helm/osccost/Chart.yaml#LL18C3-L18C3

Ok for the rest of the PR;

**Comment by albundy83** [comment]
Hello,
sure which version do you want ?  0.2.2 or 0.3.0 ?

**Comment by albundy83** [comment]
And also, as Dockerfile is updated, docker image on docker.io will be automatically updated ?
Should I change also `imageTag` value ?

**Comment by outscale-vbr** [comment]
Please also change imageTag
https://github.com/outscale/osc-cost/blob/e8554edf64d54dc6fc8e20752b8143abb7663e3d/helm/osccost/values.yaml#L43

A new release 0.3.1 will integrate your change.

https://github.com/outscale/osc-cost/blob/main/.github/workflows/release-docker-image.yml

File: helm/osccost/Chart.yaml [hunk 1]
```diff
@@ -15,10 +15,10 @@ type: application
...
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 0.2.1
+version: 0.3.0
 
 # This is the version number of the application being deployed. This version number should be
 # incremented each time you make changes to the application. Versions are not expected to
 # follow Semantic Versioning. They should reflect the version the application is using.
 # It is recommended to use it with quotes.
-appVersion: "1.16.0"
+appVersion: "0.3.0"
```
Comment (outscale-vbr) [inline helm/osccost/Chart.yaml]: 0.3.1
A new release will be necessary to build your image.

File: helm/osccost/values.yaml [hunk 2]
```diff
@@ -40,7 +40,7 @@ osccost:
     # -- Outscale provider image
       image: outscale/osc-cost
     # -- Outscale provider image tag
-      imageTag: "v0.2.0"
+      imageTag: "v0.3.0"
```
Comment (outscale-vbr) [inline helm/osccost/values.yaml]: Please set v0.3.1
A new release will be necessary to build your image.

File: helm/osccost/README.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
 # osccost
 
-![Version: 0.2.1](https://img.shields.io/badge/Version-0.2.1-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: 1.16.0](https://img.shields.io/badge/AppVersion-1.16.0-informational?style=flat-square)
+![Version: 0.3.0](https://img.shields.io/badge/Version-0.3.0-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: 0.3.0](https://img.shields.io/badge/AppVersion-0.3.0-informational?style=flat-square)
```
Comment (outscale-vbr) [inline helm/osccost/README.md]: Please set v0.3.1
A new release will be necessary to build your image.

**Comment by outscale-vbr** [comment]
Closes #101

File: helm/osccost/README.md [hunk 4]
```diff
@@ -1,6 +1,6 @@
 # osccost
 
-![Version: 0.2.1](https://img.shields.io/badge/Version-0.2.1-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: 1.16.0](https://img.shields.io/badge/AppVersion-1.16.0-informational?style=flat-square)
+![Version: 0.3.0](https://img.shields.io/badge/Version-0.3.0-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: 0.3.1](https://img.shields.io/badge/AppVersion-0.3.1-informational?style=flat-square)
```
Comment (outscale-vbr) [inline helm/osccost/README.md:3]: 0.3.1

Comment (outscale-vbr) [reply helm/osccost/README.md:3]: yes for all the line

**Comment by albundy83** [comment]
I had to change also the livenessProbe and readinessProbe as they never work

**Comment by outscale-vbr** [comment]
Ok

**Review by outscale-vbr — APPROVED** [review]
OK

**Comment by outscale-vbr** [comment]
@albundy83 it will be available with the new version 0.3.1 which will be available very soon.

**Pull request merged**